ietf-optical-impairment-topology.yang

```diff
@@ -310,19 +310,19 @@ module ietf-optical-impairment-topology {
       description "Chromatic Dispersion (CD)";
     }            
     leaf roadm-pdl {
-      type l0-types:ratio-in-db-or-null;
+      type l0-types:loss-in-db-or-null;
       description "Polarization Dependent Loss (PDL)";
     }      
     leaf roadm-inband-crosstalk {
-      type l0-types:ratio-in-db-or-null;
+      type l0-types:loss-in-db-or-null;
```

Comment:
no, inband crosstalk is not a loss, I prefer to keep ratio-in-db-or-null

Reply:
The same as above

Reply:
Addressed in https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/135/commits/62ec3365bb97f7d5b1a2b3135a70091c2f52cbc5

Co-authored-by: sergio belotti <sergio.belotti@nokia.com>